app/Community/Actions/BuildReportContextAction.php

```diff
@@ -98,4 +108,39 @@ public function execute(
 
         return $context . $messageBody;
     }
+
+    /**
+     * Build a link to a comment based on its ArticleType.
+     * Handles all comment types: user walls, game walls, achievement walls, etc.
+     */
+    private function buildCommentLink(Comment $comment): ?string
+    {
+        if (!$comment->ArticleID) {
+            return null;
+        }
+
+        $anchor = '#comment_' . $comment->ID;
+
+        return match ($comment->ArticleType) {
+            ArticleType::Game => ($game = Game::find($comment->ArticleID))
+                ? route('game.show', $game) . $anchor
+                : null,
```

Comment:
This link doesn't take me to the comment.

http://localhost:64000/game/20816#comment_4891773

For reference, this is the email notification link, which doesn't work either.

http://localhost:64000/game/20816?tab=community#comment_5700588

Reply:
Do we care about the presence/lack of `tab=community`?

Reply:
Yep, added in latest.